examples/kotlin/hello_world/MainActivity.kt

```diff
@@ -70,16 +75,23 @@ class MainActivity : Activity() {
   }
 
   private fun makeRequest(): Response {
-    return  try {
+
+    return try {
       val url = URL(ENDPOINT)
-      // Open connection to the envoy thread listening locally on port 9001
-      val connection = url.openConnection() as HttpURLConnection
-      val status = connection.responseCode
+
+      val latch = CountDownLatch(1)
```

Comment:
Alternatively, I can just run the code in the callback rather than using a countdown latch